Python/specialize.c

```diff
@@ -162,12 +162,14 @@ get_cache_count(SpecializedCacheOrInstruction *quickened) {
 static uint8_t adaptive_opcodes[256] = {
     [LOAD_ATTR] = LOAD_ATTR_ADAPTIVE,
     [LOAD_GLOBAL] = LOAD_GLOBAL_ADAPTIVE,
+    [BINARY_SUBSCR] = BINARY_SUBSCR_ADAPTIVE,
 };
 
 /* The number of cache entries required for a "family" of instructions. */
 static uint8_t cache_requirements[256] = {
     [LOAD_ATTR] = 2, /* _PyAdaptiveEntry and _PyLoadAttrCache */
     [LOAD_GLOBAL] = 2, /* _PyAdaptiveEntry and _PyLoadGlobalCache */
+    [BINARY_SUBSCR] = 1, /* _PyAdaptiveEntry */
```

Comment:
This can be zero. There is no data that needs caching.
The count can be stored in `oparg`.

Reply:
What do you mean by "the count"?

Reply:
I should have said count**er**
https://github.com/python/cpython/pull/27043/files#diff-c22186367cbe20233e843261998dc027ae5f1f8c0d2e778abfa454ae74cc59deR1931

Reply:
is oparg large enough for that?

Reply:
Same size
https://github.com/python/cpython/blob/main/Include/internal/pycore_code.h#L41

Reply:
How do we update the oparg?

Reply:
`((uint8_t *)instr)[1] = oparg;` Feel free to add an inline function or macro to [code.h](https://github.com/python/cpython/blob/main/Include/cpython/code.h)

Reply:
Should we pass a pointer to the counter to record_cache_hit, etc., instead of the cache entry?

Reply:
Pushed.   I used _Py_MAKECODEUNIT to create the new instrs, not sure a new macro would be faster?